schemas/tests/test_validate.py

```diff
@@ -0,0 +1,67 @@
+import json
+from pathlib import Path
+from functools import partial
+
+import pytest
+import jsonschema
+
+ROOT = Path(__file__).parent.parent.parent
```

Comment:
This should make it more stable I guess, although it shouldn't be an issue since it will only ever be run in a `pytest` setting, and then it will be relative to the cwd I guess? Still though, absolute paths are always nice :)
```suggestion
ROOT = Path(__file__).parent.parent.parent.resolve()
```

Reply:
I believe that `__file__` is always absolute, but no reason to not add the resolve().